cmd/policies.go

```diff
@@ -309,3 +325,345 @@ func viewPolicyCmd(ctx *cli.Context) error {
+				if err != nil {
+					// TODO: Log this!!!
+					return
+				}
+				for _, m := range members {
+					if *user.ID == *m.Body.OwnerID {
+						teamChan <- team
+						return
+					}
+				}
+			}(t)
+		}
+		wg.Wait()
+		close(teamChan)
+	}()
+}
+
+// Fetch all policies and policy-attachments for the org. The two steps are
+// independent so do them in parallel.
+func getPoliciesAndAttachments(client *api.Client, orgID *identity.ID) ([]envelope.Policy, []envelope.PolicyAttachment, error) {
+	c := context.Background()
+	wg := &sync.WaitGroup{}
+	wg.Add(2)
+
+	var pErr error
+	var policies []envelope.Policy
+	go func() {
+		defer wg.Done()
+		policies, pErr = client.Policies.List(c, orgID, "")
+	}()
+
+	var aErr error
+	var attachments []envelope.PolicyAttachment
+	go func() {
+		defer wg.Done()
+		attachments, aErr = client.Policies.AttachmentsList(c, orgID, nil, nil)
+	}()
+	wg.Wait()
+
+	if aErr != nil || pErr != nil {
+		return nil, nil, cli.NewMultiError(pErr, aErr,
+			errs.NewExitError(policyTestFailed))
+	}
+	return policies, attachments, nil
+}
+
+// Test if a Policy satisfies some condition
+type PolicyPredicate func(envelope.Policy) bool
+
+// Compound Predicate: Test of a policy satisfies all predicates
+func AllPredicate(predicates ...PolicyPredicate) PolicyPredicate {
+	return func(policy envelope.Policy) bool {
+		for _, pred := range predicates {
+			if !pred(policy) {
+				return false
+			}
+		}
+		return true
+	}
+}
+
+func filterPolicies(policies []envelope.Policy, predicate PolicyPredicate) []envelope.Policy {
+	result := make([]envelope.Policy, 0)
+	for _, pol := range policies {
+		if predicate(pol) {
+			result = append(result, pol)
+		}
+	}
+	return result
+}
+
+// Crete a Predicate to filter (exclude) policies that are not attached to any
+// of the specified teams.
+func policyAttachedToTeamsPredicate(teams []envelope.Team,
+	attachments []envelope.PolicyAttachment) PolicyPredicate {
+
+	teamsByID := make(map[identity.ID]envelope.Team)
+	for _, t := range teams {
+		teamsByID[*t.ID] = t
+	}
+
+	attachmentByPolicyID := make(map[identity.ID]envelope.PolicyAttachment, 0)
+	for _, a := range attachments {
+		attachmentByPolicyID[*a.Body.PolicyID] = a
+	}
+
+	return func(policy envelope.Policy) bool {
+		if a, ok := attachmentByPolicyID[*policy.ID]; ok {
+			if _, ok := teamsByID[*a.Body.OwnerID]; ok {
+				return true
+			}
+		}
+		return false
+	}
+}
+
+// Create a predicate to filter (exclude) policies that do not apply to the
+// specified path. A policy applies to a path if any of its resources (paths)
+// are equivalent to the specified path. The secret portion of the path is
+// ignored.
+func policyTouchesPathPredicate(pathExp *pathexp.PathExp) PolicyPredicate {
+	return func(policy envelope.Policy) bool {
+		for _, s := range policy.Body.Policy.Statements {
+			rp, err := parseRawPath(s.Resource)
+			if err != nil {
+				// TODO Log this
+				continue
+			}
+			// I'm pretty sure this is wrong...
```

Comment:
It is, but hey, you know it! :)
The pathexp pkg would need to add a Contains method to the PathExp object. That's a decent sized chunk of work, so Equality is a-ok for now.